### ISSUE TITLE
lein deps failure: honeysql

### ISSUE THREAD

**Issue by 6a68** [mozilla/datomish#15]
Tried to `lein deps`, as suggested in the README, but it failed with this error:

```
| mozilla-datomish @ 🚐   lein deps
Could not find artifact honeysql:honeysql:jar:0.7.1-SNAPSHOT in clojars (https://clojars.org/repo/)
This could be due to a typo in :dependencies or network issues.
If you are behind a proxy, try setting the 'http_proxy' environment variable.
```

What might I be missing?

**Comment by ncalexan** [comment]
This hasn't gotten upstreamed yet.  You'll need to `lein install` the branch of `honeysql` from https://github.com/rnewman/honeysql/tree/rnewman/rebase.  @rnewman can you push this along upstream?

**Comment by 6a68** [comment]
@ncalexan Aha, great, thanks.

> You'll need to lein install the branch of honeysql from https://github.com/rnewman/honeysql/tree/rnewman/rebase

I'm not too familiar with Leiningen; could you explain how I might do this in a little more detail?

**Comment by ncalexan** [comment]
I think you'll need to clone that branch, and then just `lein install` from the root.  That'll install the snapshot JAR into your local Maven repository, ready for Datomish to find it.

**Comment by rnewman** [comment]
Exactly that. Yeah, I'll push on it.

**Comment by rnewman** [comment]
Waiting on https://github.com/jkk/honeysql/pull/130 .

**Comment by 6a68** [comment]
@ncalexan Perfect, thanks

**Comment by rnewman** [comment]
Fixed by using `honeysql` 0.8.
